### PR TITLE
Fix voice conversion inference

### PR DESCRIPTION
**Pull request by Edresson**
I fixed some issues with the voice conversion inference and added a unit test to guarantee that it will not break in the future.

### CODE REVIEW

File: TTS/tts/models/vits.py [hunk 1]
```diff
@@ -1127,12 +1127,12 @@ def inference_voice_conversion(
...
             self.config.audio.win_length,
             center=False,
-        ).transpose(1, 2)
+        )
         y_lengths = torch.tensor([y.size(-1)]).to(y.device)
         speaker_cond_src = reference_speaker_id if reference_speaker_id is not None else reference_d_vector
         speaker_cond_tgt = speaker_id if speaker_id is not None else d_vector
         # print(y.shape, y_lengths.shape)
-        wav, _, _ = self.voice_conversion(y, y_lengths, speaker_cond_src, speaker_cond_tgt)
+        wav, _, _ = self.voice_conversion(y.transpose(1, 2), y_lengths, speaker_cond_src, speaker_cond_tgt)
```
Comment (p0p4k) [inline TTS/tts/models/vits.py:1135]: Just remove the transpose(1,2) from lines 1135 and 1160.
I believe it is redundant.

Comment (p0p4k) [inline TTS/tts/models/vits.py:1135]: Just remove the transpose(1,2) from lines 1135 and 1160.
I believe it is redundant.

Comment (Edresson) [reply TTS/tts/models/vits.py:1135]: Agreed, good catch :). Thanks for the feedback.

**Review by WeberJulian — APPROVED** [review]
Thanks for the PR, I checked and it works.

**Pull request merged**